src/routes/routes/[id]/+page.svelte

```diff
@@ -0,0 +1,80 @@
+<script lang="ts">
+	import { Map, Marker, controls } from '@beyonk/svelte-mapbox';
+	import type { CurrentPosition } from '$lib/getCurrentPosition.js';
+	import { onMount } from 'svelte';
+	import { env } from '$env/dynamic/public';
```

Comment:
`import { PUBLIC_MAPBOX_API_KEY } from "$env/static/public"`

could be better typesafe use of env vars.

Reply:
nice, didn't know about this.